src/evolution/evolutionary_strategy.py

```diff
@@ -109,6 +109,8 @@ def run(self):
                 self.step()
                 self.update_log(istep)
 
+            self.population.perform_visualizations()
```

Comment:
I think this should probably be a script that gets ran after the program, rather than hard coded like this. But in any case, it should not be hardcoded in the base evolutionary strategy (especially since this `run` method is likely to get overridden).